eng/sdk/sync_sdk.py

```diff
@@ -71,16 +71,7 @@ def update_sdks():
         generated_test_exists = os.path.isdir(generated_test_path)
 
         logging.info('Generate for module %s', artifact)
-
-        cmd = [
-            'pwsh',
-            os.path.join(sdk_root, 'eng/common/scripts/TypeSpec-Project-Sync.ps1'),
-            module_path
-        ]
-        subprocess.check_call(cmd, cwd=sdk_root)
-
-        cmd[1] = os.path.join(sdk_root, 'eng/common/scripts/TypeSpec-Project-Generate.ps1')
-        subprocess.check_call(cmd, cwd=sdk_root)
+        subprocess.check_call(['tsp-client', 'update'], cwd=module_path)
```

Comment:
Is there document about what does `tsp-client update` do? to update typespec dependencies to latest version and generate sdk?

Reply:
https://github.com/Azure/azure-sdk-tools/tree/main/tools/tsp-client

Reply:
BTW, I've updated Quickstart as well https://github.com/Azure/azure-sdk-for-java/wiki/TypeSpec-Java-Quickstart#use-typespec-defined-in-rest-api-specifications

Reply:
ok, I might miss some context. What is the relationship between nightly sdk preview pipeline and this PR's `tsp-client update`?

Reply:
Nothing related at all.

tsp-client just replaces script `TypeSpec-Project-*` https://github.com/Azure/azure-sdk-tools/tree/main/eng/common/scripts

Reply:
ok, I see.